.circleci/config.yml

```diff
@@ -108,11 +108,13 @@ workflows:
             branches:
               only: master
```

Comment:
How about adding ignore here too, so that it doesn’t release every single commit to master?

Reply:
did that after reading all convo in gitter :-)